src/Breadcrumb/index.jsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+import styles from './Breadcrumb.scss';
+import Icon from '../Icon';
+
+const Breadcrumbs = ({
+  links, activeLabel, spacer, clickHandler,
+}) => {
+  const linkCount = links.length;
```

Comment:
It feels cleaner to me to just use links.length instead of assigning it to a variable for just one use

Reply:
It's more performant to assign it to a variable, otherwise it has to calculate `.length` for each loop of `map`.